development/Vagrantfile.example

```diff
@@ -1,20 +1,72 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-BOX = ENV.fetch('BOX', 'ubuntu-12.04')
+require 'pry'
 
-Vagrant.require_plugin 'vagrant-proxyconf'
+BOX = ENV.fetch('BOX', 'ubuntu/bionic64')
+DISABLE_SHARE = false
```

Comment:
This changes the old behaviour. If that is wanted, the comment where this is used should be updated accordingly.

Reply:
I needed to remove the `require_plugin` line because it was deprecated some time ago in favor of using the Gemfile.

- https://stackoverflow.com/questions/19492738/demand-a-vagrant-plugin-within-the-vagrantfile
- https://github.com/hashicorp/vagrant/issues/4349

I will add a note to the Gemfile.

Reply:
Oh, sorry, I was unclear. I meant the value of `DISABLE_SHARE`. Earlier it defaulted to `true`.

Reply:
> I needed to remove the `require_plugin` line because it was deprecated some time ago in favor of using the Gemfile.

That was not needed for the testing nor loading the plugin. It was in the example Vagrantfile to confirm that the plugin is installed. Kind of using a side effect. So `if Vagrant.has_plugin?('vagrant-proxyconfg')` would be the replacement, but I'm fine leaving it off.

Reply:
Ahh, thanks for clarifying and no worries.  I think I added `DISABLE_SHARE` because I was testing docker provider in Vagrant and it didn't like the share being enabled.  

fixed via: 3fa63e55f7bb70cc9e313f5911c20689d938c093